engine/src/main/java/org/terasology/engine/rendering/opengl/OpenGLTexture.java

```diff
@@ -82,7 +82,8 @@ protected void doReload(TextureData data) {
                                         resources.graphicsManager.disposeTexture(newId);
                                     } else {
                                         resources.id = newId;
-                                        logger.debug("Bound texture '{}' - {}", getUrn(), resources.id);
+                                        logger.atDebug().addArgument(() -> getUrn()).addArgument(resources.id).
+                                                log("Bound texture '{}' - {}");
```

Comment:
Simple getter method so impact neglectable
```suggestion
                                        logger.debug("Bound texture '{}' - {}", getUrn(), resources.id); //NOPMD
```